RELEASE_PROCESS.rst

```diff
@@ -0,0 +1,39 @@
+Prepare for the Release
+-----------------------
+
+Clone a fresh copy from the root repo. Do not attempt a release in your
+current working repository as the following commands expect a fresh clone.
+
+::
+
+     git clone --branch master git@github.com:juju/juju-gui.git
+     cd juju-gui
+
+Increment the version number using the ``make bumpversion`` target.  If you
```

Comment:
You could also add that this will make a git commit for you so make sure that this clone has the proper user.name and user.email values set.